db/migrate/20190218180413_move_tag_value_to_tags.rb

```diff
@@ -0,0 +1,40 @@
+class MoveTagValueToTags < ActiveRecord::Migration[5.2]
+  def up
+    add_column :tags, :value, :string, :null => false, :default => ""
+
+    remove_column :container_group_tags,    :value
```

Comment:
and here we're just removing the index (not sure if this line removes it), but we need to add the new unique index that will be on `["container_group_id", "tag_id"]` and same for others
`

Reply:
Yeah it looks like there was only an index on the individual columns, probably should be on the pair of columns.

Reply:
the indexed looked like 
```t.index ["container_node_id", "tag_id", "value"], name: "uniq_index_on_container_node_id_tag_id_and_value", unique: true```

the remove column is probably removing it

Reply:
Interesting okay there was a separate one then just on tag_id then also, `t.index ["tag_id"], name: "index_container_group_tags_on_tag_id"`